docs/src/utils/getCustomStaticPath.ts

```diff
@@ -0,0 +1,14 @@
+import { FRAMEWORKS } from '@/data/frameworks';
+
+export const getCustomStaticPath = (platforms: string[] | string) => {
+  const platformsArr = Array.isArray(platforms)
```

Comment:
nit: nested ternaries are a bit hard to read, can you use if statements here instead?